src/browser/apis/notes_api.js

```diff
@@ -66,10 +66,10 @@ var NotesAPI = (function() {
   // Public API
   return {
     saveNote: function(note, cb) {
-      note.attributes._id = note.attributes._id || utils.createGuid();
-      var filename = api.noteFilename(note.attributes._id);
+      note._id = note._id || utils.createGuid();
+      var filename = api.noteFilename(note._id);
       api.findNote(filename, function() {
-          api.writeNote(filename, note.attributes, note, cb);
+          api.writeNote(filename, note, note, cb);
```

Comment:
can probably refactor out one of these note objects and use a temp notedata variable down the cascade.